iroh/src/discovery.rs

```diff
@@ -833,27 +834,6 @@ mod test_dns_pkarr {
         Ok(())
     }
 
-    #[tokio::test]
```

Comment:
why is this removed?

Reply:
That test is identical to the test above.